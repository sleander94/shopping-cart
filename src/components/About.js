import React from 'react';
import github from '../images/iconmonstr-github-1.svg';
import instagram from '../images/iconmonstr-instagram-11.svg';
import facebook from '../images/iconmonstr-facebook-1.svg';
import twitter from '../images/iconmonstr-twitter-1.svg';
import '../styles/About.css';

const About = () => {
  return (
    <div id="about">
      <div className="icons">
        <img src={github} alt="github"></img>
        <img src={instagram} alt="instagram"></img>
        <img src={facebook} alt="facebook"></img>
        <img src={twitter} alt="twitter"></img>
      </div>
    </div>
  );
};

export default About;
