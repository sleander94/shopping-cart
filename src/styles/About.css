#about {
  display: grid;
  justify-items: center;
  align-items: center;
}

#about .icons {
  display: flex;
  justify-content: space-between;
  width: 40%;
}

#about .icons img {
  height: 100px;
  width: 100px;
}
