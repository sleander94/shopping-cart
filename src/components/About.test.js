import React from 'react';
import renderer from 'react-test-renderer';
import '@testing-library/jest-dom';
import About from './About';

describe('About component', () => {
  it('renders correctly', () => {
    const about = renderer.create(<About />);
    expect(about).toMatchSnapshot();
  });
});
