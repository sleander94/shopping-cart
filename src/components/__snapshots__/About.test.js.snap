// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component renders correctly 1`] = `
<div
  id="about"
>
  <div
    className="icons"
  >
    <a
      href="https://github.com/sleander94/"
    >
      <img
        alt="github"
        src="iconmonstr-github-1.svg"
      />
    </a>
    <a
      href="https://www.youtube.com/watch?v=dQw4w9WgXcQ"
    >
      <img
        alt="instagram"
        src="iconmonstr-instagram-11.svg"
      />
    </a>
    <a
      href="https://c.tenor.com/GbutACtJBTMAAAAd/cat.gif"
    >
      <img
        alt="facebook"
        src="iconmonstr-facebook-1.svg"
      />
    </a>
    <a
      href="https://www.youtube.com/watch?v=dQw4w9WgXcQ"
    >
      <img
        alt="twitter"
        src="iconmonstr-twitter-1.svg"
      />
    </a>
  </div>
</div>
`;
